### PR TITLE
Lathropd patch 2

### PR DESCRIPTION
**Pull request by lathropd**
Update some public-facing perl6 references

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -5,7 +5,7 @@ by synopsis (when applicable) or put them in the OVERALL section at the top
 if they apply to the test suite in general.
 
 NOTE that references to a t/ directory are for the t/ directory in the mu
-repository found here:  https://github.com/perl6/mu
+repository found here:  https://github.com/raku/mu
```
Comment (JJ) [inline TODO:8]: Maybe capitalize that?

File: TODO [hunk 2]
```diff
@@ -16,7 +16,7 @@ commit message:
 [TODO]: Added more tasks for S02.
 
 This way, people working on the test suite can easily spot these commits
-in #perl6.
+in #raku.
```
Comment (JJ) [inline TODO:19]: Not totally clear what the hashtag is about. If it's a hashtag, in Twitter #rakulang is used. And apparently in Rosettacode it should be capitalized.

File: docs/announce/6.e.md [hunk 3]
```diff
@@ -32,10 +32,10 @@ This ChangeLog is targeted towards language users, to help with preparation to u
 compilers supporting latest language version. Thus, it does not contain every minute
 change to the specification that occurred. Implementations wishing to ensure full
 compliance with the new version of the language specification should execute the
-test suite available at https://github.com/perl6/roast/ and examine any failing tests.
+test suite available at https://github.com/raku/roast/ and examine any failing tests.
```
Comment (JJ) [inline docs/announce/6.e.md:36]: Again, capitalizing is probably better. It's redirected, but still it's better to use like it should.

File: TODO (hunk 2 above)
Comment (Altai-man) [reply TODO:19]: I think this is about irc channel, though.

File: docs/announce/6.e.md (hunk 3 above)
Comment (lathropd) [reply docs/announce/6.e.md:36]: Fixed the capitalization. Thanks!

**Pull request merged**